configs/argodev/main.yaml

```diff
@@ -39,6 +39,27 @@ services:
   authentication_service:
     enabled: true
     dev_auth_enabled: true
+    modes:
+      - type: internal
+        enabled: true
+        public_registration_enabled: true
+      - type: openid
+        enabled: true
+        openid:
+          id: keycloak
+          chorus_backend_host: "https://backend.dev.chorus-tre.ch"
+          authorize_url: "https://auth.dev.chorus-tre.ch/realms/chorus/protocol/openid-connect/auth"
+          token_url: "https://auth.dev.chorus-tre.ch/realms/chorus/protocol/openid-connect/token"
+          user_info_url: "https://auth.dev.chorus-tre.ch/realms/chorus/protocol/openid-connect/userinfo"
+          logout_url: "https://auth.dev.chorus-tre.ch/realms/chorus/protocol/openid-connect/logout?redirect_uri=https://backend.dev.chorus-tre.ch"
```

Comment:
You should really use https://auth.dev.chorus-tre.ch/realms/chorus/.well-known/openid-configuration instead.

Reply:
e.g. https://oauth2-proxy.github.io/oauth2-proxy/configuration/providers/keycloak_oidc/ has a `oidc-issuer-url` which infers that `.well-known` endpoint to fetch everything it needs.